newapi/ooniapi/probe_services.py

```diff
@@ -41,6 +43,27 @@ def generate_report_id(test_name, cc: str, asn_i: int) -> str:
     return rid
 
 
+def get_probe_ip() -> str:
+    real_ip_headers = [
+        "X-Forwarded-For",
+        "X-Real-IP"
+    ]
+    for h in real_ip_headers:
```

Comment:
Maybe it's worth unrolling the loop for only 2 entries

Reply:
That's what I had originally done, but then I figured we might need to eventually enrich this list to include more header values to support more caching proxies. I haven't, for example, tested what headers are used by cloudfront or other caching proxies we might have interest in using like haproxy.